python/src/opendp/_lib.py

```diff
@@ -124,6 +125,30 @@ def next_raw(_voidp):
 except ImportError:  # pragma: no cover
     pass
 
+try:
+    pl = import_optional_dependency("polars")
+    from polars.plugins import register_plugin_function
```

Comment:
I'd prefer to keep the `except ImportError` out of the code, now that it's cleaner.